streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -86,13 +88,14 @@
      */
     public <K, V> KStream<K, V> stream(final AutoOffsetReset offsetReset,
                                        final String... topics) {
-        return stream(offsetReset, null, null, topics);
+        return stream(offsetReset, null, null, null, topics);
     }
 
+
     /**
      * Create a {@link KStream} from the specified topic pattern.
-     * The default {@code "auto.offset.reset"} strategy and default key and value deserializers as specified in the
-     * {@link StreamsConfig config} are used.
+     * The default {@link TimestampExtractor}, default {@code "auto.offset.reset"} strategy and default key and value
```

Comment:
Nit: Please use the same order in all JavaCode -- above TimestampExtractor is second -- I don't care which order, but please be consistent. Maybe follow parameter order of the method overload that provides all parameters ?